app/views/community_memberships/_confirmation_form.haml

```diff
@@ -2,11 +2,14 @@
   initialize_confirmation_pending_form("#{I18n.locale}","#{email_not_accepted_message}");
 
 - email_to_confirm = @current_user.latest_pending_email_address(@current_community)
+- contact_support_link = link_to t("sessions.confirmation_pending.contact_support_link_text"), "mailto:#{support_email}", "data-uv-trigger" => "contact"
```

Comment:
To my understanding, it will trigger the UserVoice widget, see the screenshot

![screen shot 2016-10-31 at 15 54 01](https://cloud.githubusercontent.com/assets/429876/19856486/529b3032-9f82-11e6-99bf-5f389154a81d.png)